lib/client.js

```diff
@@ -69,10 +67,18 @@ var OLM_ALGORITHM = "m.olm.v1.curve25519-aes-sha2";
  * {@link requestFunction} for more information.
  *
  * @param {string} opts.accessToken The access_token for this user.
+ *
  * @param {string} opts.userId The user ID for this user.
- * @param {Object} opts.store Optional. The data store to use. If not specified,
+ *
+ * @param {Object=} opts.store The data store to use. If not specified,
  * this client will not store any HTTP responses.
  *
+ * @param {string=} opts.deviceId A unique identifier for this device, for use
+ *    in end-to-end crypto. If not specified, end-to-end crypto will be disabled.
+ *
+ * @param {Object=} opts.sessionStore A store to be used for end-to-end crypto
+ *    session data. If not specified, end-to-end crypto will be disabled.
+ *
```

Comment:
Ideally it would be some interface which WebStorageSessionStore implements, and which I can't be bothered to write. But ok, I'll make this clearer.